src/hardware/sblaster.cpp

```diff
@@ -70,6 +71,15 @@ enum SB_TYPES {
 	SBT_GB   = 7
 };
 
+enum FILTER_TYPES {
+	FT_NONE,
+	FT_SB1,
+	FT_SB2,
+	FT_SBPRO1,
+	FT_SBPRO2,
+	FT_SB16,
+};
```

Comment:
Cool, didn't know that about enum classes. I was just following the patterns in this file (it seemed something really old and relatively untouched), but if we're fine with gradually improving it, I'm all for it.

Reply:
Right on. Yes, whenever Staging adds code, we try to upgrade the syntax (to C++11/14/17) or safer constructs (strings/string_views/containers/etc.. instead of the C styles) - at least in the lines we're touching.